### PR TITLE
Feature/property code inclusion

### PR DESCRIPTION
**Pull request by jasebell**

### CODE REVIEW

File: src/kixi/hecuba/weather/service.clj [hunk 1]
```diff
@@ -29,13 +30,14 @@
                         (kafka/message kafka-topic
                                        (.getBytes (gen-message
                                                    (get entity "entity_id")
+                                                   (get entity "property_code")
                                                    entity-type
                                                    entity-action))))))
 (defn run-api-search [args-map]
```
Comment (elisehuard) [inline src/kixi/hecuba/weather/service.clj]: worth it destructuring the args-map to lighten the code? #nitpick

**Comment by elisehuard** [comment]
mostly lgtm :shipit:

File: src/kixi/hecuba/weather/service.clj (hunk 1 above)
Comment (jasebell) [inline src/kixi/hecuba/weather/service.clj]: #Nitpick is good, means there's something I can learn and improve on.

**Comment by elisehuard** [comment]
:shipit:

**Pull request merged**